autoreact/ktp_xarray/xarray_wrappers.py

```diff
@@ -3,12 +3,11 @@
 """
 
 import xarray
+import numpy
 
 # Constructors
 def from_data(temps, press, rates):
-    """
-    Construct a KTP DataArray from data
-    """
+    """Construct a KTP DataArray from data"""
 
     ktp = xarray.DataArray(rates, (("pres", press), ("temp", temps)))
```

Comment:
It would be good here to first cast the values to specific datatypes for consistency:
```
rates = numpy.array(rates, dtype=float)
press = numpy.array(press, dtype=float)
temps = numpy.array(temps, dtype=float)
```